cmd/crio/config.go

```diff
@@ -131,6 +131,13 @@ image_volumes = "{{ .ImageVolumes }}"
 insecure_registries = [
 {{ range $opt := .InsecureRegistries }}{{ printf "\t%q,\n" $opt }}{{ end }}]
 
+# registries is used to specify registries to be used when pulling an unqualified
+# image (e.g. fedora:rawhide). "docker.io" is left here as a default registry.
+# Users can remove it and use whatever default registry they need.
+registries = [
+  "docker.io",
```

Comment:
What does
$ sudo ./crio --config test.config --registry test.rhel.com | grep docker.io
Show?
What does the list of registries show?  It should show docker.io and test.rhel.com from my understanding of the code.  But if I remove docker.io from test.config and run the same test does it show just test.rhel.com and NO docker.io?  That is the problem, but it is probably not a big deal since the config command is probably only run in rpm.